inc/TRestRawSignalGeneralFitProcess.h

```diff
@@ -39,12 +39,12 @@ class TRestRawSignalGeneralFitProcess : public TRestEventProcess {
     void Initialize();
 
     void LoadDefaultConfig();
-    
+
     TVector2 fFunctionRange = TVector2(0, 0);
     string fFunction;
-    
-    TF1* f;
```

Comment:
@DavidDiezIb I changed this to follow the naming convention --> f + first letter uppercase keyword. Also, we are not interested to write the object to disk, so I added `//!`.